File: L1/Models/Grammar.cs
using System.ComponentModel.DataAnnotations;

namespace L1.Models
{
    public  class Grammar
    {
        [Key]
        public int Id { get; set; }

        public string Name { get; set; }
        public string Description { get; set; }
        public string  Image { get; set; }
        

   //     public abstract void playSound();

        public bool isSameAs(int id)
        {
            return this.Id == id;
        }
    }
}

File: IOS app/Learning/Model/Grammar.swift
//
//  Grammar.swift
//  Learning
//
//  Created by Marjus Nielsen on 2022-05-03.
//

import Foundation


struct Grammar: Hashable, Codable, Identifiable{ 
    var id: Int
    var name: String
    var description: String
    var image: String
    
    func isSameAs(_ grammar: Grammar) -> Bool {
        self.id == grammar.id
    }
}


